pkgs/tools/archivers/star/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "star-${version}";
+  version = "1.5.3";
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/s-tar/files/${name}.tar.gz";
+    sha256 = "1yii6p6x38r6w12s5yrdi08aij481jcg3xf8g738nmh0hqypbfl7";
+  };
+
+  preConfigure = "rm configure";
+  preBuild = "sed 's_/bin/__g' -i RULES/*";
+  makeFlags = [ "DESTDIR=$(out)" "INS_BASE=/ install"];
+  postInstall = ''
+  find $out/bin -type l -delete
+  rm -r $out/etc $out/include $out/sbin
+  '';
+
+  meta = {
+    description = "Star is a very fast tar like tape archiver with  improved functionality.";
```

Comment:
No need to start the description with the package name and it should not end with a period. Also there is a double space here.

Perhaps change to
```
description = "A very fast tar like tape archiver with improved functionality";
```